### PR TITLE
Contact us page

### PR DESCRIPTION
**Pull request by danielwilliams94**
Styling to handle contact page

image

### CODE REVIEW

File: src/components/Contentful/sass/components/_contentful_grid.scss [hunk 1]
```diff
@@ -84,3 +84,12 @@
...
 }
+
+.contentful-grid.black {
+  background-color: #333333;
+  margin-bottom: 0;
+
+  .prose {
+    color: #ffffff;
+  }
+}
```
Comment (mmmmillar) [inline src/components/Contentful/sass/components/_contentful_grid.scss]: I think these colours for black grids should already be covered in _new_design_overrides.scss

File: src/components/Contentful/Grid.js [hunk 2]
```diff
@@ -254,23 +265,27 @@ function GridComponentArrayRenderer({
...
+            smallColumnWidth={contentArr[0].smallColumnWidth}
+            smallColumnOffset={contentArr[0].smallColumnOffset}
+            key={i}
+          >
+            {contentArr.map((content, indexKey) => (
+              <GridComponentRenderer key={indexKey} {...content} />
+            ))}
+          </GridCol>
+        )
+      })}
```
Comment (mmmmillar) [inline src/components/Contentful/Grid.js]: I know I called it contentArr but could we rename it to "group" or similar for readability?

File: src/components/Contentful/sass/sections/_contact.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.contact {
+    b {
+        font-family: 'poppins';
+        color: $mt-green;
+    }
+
+    .contact__blurb {
+        p {
+            font-family: 'poppins';
```
Comment (mmmmillar) [inline src/components/Contentful/sass/sections/_contact.scss]: Should be "font-family: $poppins;"

File: src/components/Contentful/sass/sections/_contact.scss [hunk 4]
```diff
@@ -8,27 +8,57 @@
...
+            line-height: 1.5;
+
+            b {
+                font-size: $px24;
+            }
 
-            p {
-                margin-bottom: 14px;
+            a {
+                font-family: 'poppins';
```
Comment (mmmmillar) [inline src/components/Contentful/sass/sections/_contact.scss]: Should be "font-family: $poppins;"

File: src/components/Contentful/sass/components/_contentful_grid.scss (hunk 1 above)
Comment (danielwilliams94) [reply src/components/Contentful/sass/components/_contentful_grid.scss]: That's what I thought but I couldn't see them anywhere - I might have missed it though, I'll have a look

Comment (danielwilliams94) [reply src/components/Contentful/sass/components/_contentful_grid.scss]: You're correct - I think I hadn't tagged the page as new-design before and that's why I missed it. Removed.

**Pull request merged**